data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -91,7 +105,7 @@ public static void main(String[] args) throws IOException, ParseException,
 //			lineageMain.query = LineageConstants.query;
 			System.out.println("Error: Invalid inputs for LineageMain");
 			throw new Exception("Invalid inputs for LineageMain");
-		}
+		}*/
 
 		lineageMain.getLineageInfo();
 //		lineageMain.generateOperatorTree(new HiveConf(), query);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)